### DISCUSSION TITLE
Bluetooth paired success but Iosender cannot connect the com port

### DISCUSSION THREAD

**Discussion by jal123sin** [terjeio/ioSender#303] [Q&A] [answered]
Hello, I can pair the grblhal and I can see the bluetooth settings tag as the picture shown. But Iosender connection tag cannot see the paired com port. Do I miss something to set it？Please advise a correct way to solve it.
![pic7](https://github.com/terjeio/ioSender/assets/136598254/5d7567d7-33de-4419-9bef-2322d089b79e)
![pic8](https://github.com/terjeio/ioSender/assets/136598254/29b67cb0-43b3-45f6-b9bb-6a2502c08c6b)

**Comment by terjeio** [comment]
Which controller/Bluetooth hardware are you connecting to?
I cannot find the _Bluetooth Settings_ dialog on my Win10 machine, however this is how it shows up in the _Device and Pinters_ properties:

![image](https://github.com/terjeio/ioSender/assets/20260062/e9475db5-902d-42b3-9d9a-a6b41ad1f5a5)

There is only one port associated, you have two? This could be a reason for why it is not enumerating?

**Reply by jal123sin** [reply]
My system is win10, bluetooth controller is a usb type dongle type it shown version 4.0 , MKS DLC32 2.0 board and I use grblhal web builder to download the latest version. I think you are right there are two ports appeared was the problem. So, would you please to help me find out why this happened.

**Reply by terjeio** [reply]
I've now checked with an ESP32 controller and I have no problem connecting to it via Bluetooth.
Could it be that you have a driver issue? This is the driver that my dongle uses:

![image](https://github.com/terjeio/ioSender/assets/20260062/d549a98d-37a0-459d-a3f5-e02c5b70e068)

Can you try connecting with a different computer, preferably a laptop with inbuilt Bluetooth, or a phone?

BTW the settings (baud rate etc) are overridden by the sender so they do not matter.

**Reply by jal123sin** [reply, marked as answer]
Hello, I tested it with an older win7 pc. It can show as your display, there is only one SPP com port appear with grblhal device and it running well now, thank for your support.
